modules/ROOT/pages/storage.adoc

```diff
@@ -54,10 +55,11 @@ storage:
   - device: /dev/vda
     wipe_table: false
     partitions:
+    - number: 4
```

Comment:
I'm a little uneasy about the tradeoff here, that we're using better syntax in exchange for a hardcoded reference to partition 4.  Should we also reference `label: root`?

Reply:
> I'm a little uneasy about the tradeoff here, that we're using better syntax in exchange for a hardcoded reference to partition 4.

Yeah, it's not great. Would've been nice if `label: root` alone actually identified the specific partition instead of creating a separate one. (Thoughts on tweaking the next spec 3 subversion to do this? I'm guessing it'd be hard because it affects merge semantics.)

I think I do still prefer doing it this way because it's IMO much easier for users to understand what's going on. Otherwise, it all feels very indirect and we have to talk about offsets and are exposed to the fact that the sizes of the first three partitions affects the actual size of the rootfs. Could probably be convinced otherwise though. :)  I can add the `label: root` as a sanity-check.

Reply:
Offhand I'm not sure why `label` isn't defined that way.  It may be relevant that the UEFI spec apparently doesn't require the label to be unique.  (Of course, we make that assumption elsewhere, so perhaps we could do so in Ignition too.)  In any event, I think it'd require a major spec version bump to change this.